### PR TITLE
Add sources plugin to Maven build

### PR DESCRIPTION
**Pull request by dsyer**
It's nice to have sources in the IDE.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -165,6 +165,20 @@
                     </execution>
                 </executions>
             </plugin>
+            <!-- Sources jar -->
+            <plugin>
```
Comment (slinkydeveloper) [inline pom.xml:169]: We already have this in the release profile, do we need it 2 times?

**Review by slinkydeveloper — COMMENTED** [review]
Thanks for your contribution, I have just a small comment about this

File: pom.xml (hunk 1 above)
Comment (dsyer) [reply pom.xml:169]: I didn't notice, sorry. Why would you want it in a profile though? If I just `mvn clean install` locally, I would want to see sources. Can I remove it from the profile?

Comment (slinkydeveloper) [reply pom.xml:169]: I think you can remove it from the profile. The release profile is used both for snapshots and production releases

Comment (slinkydeveloper) [reply pom.xml:169]: Ok as is now it's fine

**Pull request merged**